sanity/doctors/ParcelDoctor.js

```diff
@@ -172,4 +181,12 @@ export class ParcelDiagnosis extends Diagnosis {
       await this.replayEvents(events)
     }
   }
+
+  getEstateIds() {
+    return new Set(
```

Comment:
I think we should just return an array here, because we normally don't use Sets in the rest of the codebase and we're creating an array when we use it anyways

Reply:
I don't mean that doing a distinct here is wrong, just that we can do something like:

```typescript
const estateIds = new Set()

for (const ({currentEstateId}) of this.faultyParcels) {
  if (currentEstateId) {
    estateIds.add(parseInt(currentEstateId, 10)) 
  }
}

return Array.from(estateIds)
```